Dockerfile

```diff
@@ -1,145 +1,157 @@
 #--------------------------------------
 # Target image to build
 #--------------------------------------
-ARG IMAGE=latest
+ARG TARGET=latest
 
 #--------------------------------------
 # Ubuntu base image to use
 #--------------------------------------
-ARG FLAVOR=latest
-
-#--------------------------------------
-# renovate rebuild trigger
-#--------------------------------------
-FROM renovate/ubuntu@sha256:b20bd58a962952580352f731606c63f621e525cc4706a4b78baf71e852b69da4 as base-latest
-FROM renovate/ubuntu:bionic@sha256:1ab4cb684944bfd76b2a29c368e4e221b796f08fb826c1a826327476cf196647 as base-bionic
-FROM renovate/ubuntu:focal@sha256:b20bd58a962952580352f731606c63f621e525cc4706a4b78baf71e852b69da4 as base-focal
+# renovate: datasource=docker
+ARG BASE_VERSION=ubuntu:bionic@sha256:ea188fdc5be9b25ca048f1e882b33f1bc763fb976a8a4fea446b38ed0efcbeba
+ARG USER_NAME=ubuntu
+ARG USER_ID=1000
+ARG APP_ROOT=/usr/src/app
 
 #--------------------------------------
 # Image: base
 #--------------------------------------
-FROM base-${FLAVOR} as base
+FROM ${BASE_VERSION} as base
 
-ARG BUILDPACK_VERSION
-LABEL org.opencontainers.image.source="https://github.com/renovatebot/docker-buildpack" \
-  org.opencontainers.image.version="${BUILDPACK_VERSION}"
+ARG USER_NAME
+ARG USER_ID
+ARG APP_ROOT
 
-USER root
-
-# Zombie killer: https://github.com/Yelp/dumb-init#readme
-RUN apt-get update && \
-  apt-get install -y dumb-init && \
-  rm -rf /var/lib/apt/lists/*
+LABEL maintainer="Rhys Arkins <rhys@arkins.net>" \
+  org.opencontainers.image.source="https://github.com/renovatebot/docker-buildpack"
 
 # loading env
 ENV BASH_ENV=/usr/local/etc/env
 SHELL ["/bin/bash" , "-c"]
 
-ENTRYPOINT [ "docker-entrypoint.sh" ]
+ENTRYPOINT ["/bin/bash" , "-c", "docker-entrypoint.sh" ]
```

Comment:
Should these lines maybe be done after the COPY command, i.e. so that they exist first? Doesn't *really* matter but would be more correct?